### PR TITLE
new blog post on gradio-blocks

### PR DESCRIPTION
**Pull request by abidlabs**

### CODE REVIEW

File: gradio-blocks.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+</div>
+
+### Machine Learning Demos 
+
+Machine learning demos are an increasingly vital part of releasing a model. Demos allow anyone — not just ML engineers — to try out a model in the browser, give feedback on predictions, and build trust in the model if it performs well. 
+
+More than 600,000 ML demos have been built with the Gradio library since its first version in 2019, and today, we are thrilled to announce **Gradio 3.0**: a grounds-up redesign of the Gradio library 🥳
+
+
+### What's New in Gradio 3.0?
```
Comment (dawoodkhan82) [inline gradio-blocks.md:32]: Why not start with the most important things first.
1. blocks
2. redesign
3. other small misc changes (dataframe etc.)

File: gradio-blocks.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+🔥 A complete redesign of the frontend
```
Comment (dawoodkhan82) [inline gradio-blocks.md:34]: Emphasize that we have taken all of the feedback we have been hearing from our developers to create this new look/feel

File: gradio-blocks.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+<img class="max-w-full mx-auto my-6" style="width: 54rem" src="/blog/assets/68_gradio_blocks/flipper.png">
+
+For a step-by-step introduction to Blocks, check out [the dedicated Blocks Guide](https://www.gradio.app/introduction_to_blocks/)
+
+### The Gradio Blocks Party 
+
+We're very excited about **Gradio Blocks** -- and we'd love for you to try it out. Even if you've never used Gradio before, this is the perfect time to start, because we are throwing a Blocks Party (😉) until the end of May. We'll be giving out prizes at the end of the Party for demos built using Blocks.
+
+Learn more about Blocks Party here: https://huggingface.co/spaces/Gradio-Blocks/README
+
```
Comment (dawoodkhan82) [inline gradio-blocks.md:109]: Do we want to mention the course. Or maybe just the blocks section?

**Review by dawoodkhan82 — DISMISSED** [review]

File: gradio-blocks.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+Once you run `launch()`, the following demo runs on a port:
+
+<img class="max-w-full mx-auto my-6" style="width: 54rem" src="/blog/assets/68_gradio_blocks/flipper.png">
+
+For a step-by-step introduction to Blocks, check out [the dedicated Blocks Guide](https://www.gradio.app/introduction_to_blocks/)
+
+### The Gradio Blocks Party 
+
+We're very excited about **Gradio Blocks** -- and we'd love for you to try it out. Even if you've never used Gradio before, this is the perfect time to start, because we are throwing a Blocks Party (😉) until the end of May. We'll be giving out prizes at the end of the Party for demos built using Blocks.
```
[the remark this answers is not in the document]
Comment (abidlabs) [reply gradio-blocks.md:106]: Well technically everyone gets a badge..

[the remark this answers is not in the document]
Comment (abidlabs) [reply gradio-blocks.md:106]: Oh yeah you're right, wow I'm sleep deprived

File: gradio-blocks.md (hunk 1 above)
Comment (abidlabs) [reply gradio-blocks.md:32]: I figured blocks naturally led into the blocks party?

File: gradio-blocks.md (hunk 3 above)
Comment (abidlabs) [reply gradio-blocks.md:109]: We'll announce that formally tomorrow, so I think fine to wait

**Review by nateraw — CHANGES_REQUESTED** [review]
TODOs:

- [x] Update _blog.yml so the post actually shows up in the UI ( I will do this rn)
- [ ] Update thumbnail (current one is set to be an old one)
- [ ] Make mindseye-lite.png smaller (convert to jpg/scale down size or something)

File: gradio-blocks.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+### What's New in Gradio 3.0?
+
+🔥 A complete redesign of the frontend, based on the feedback we're hearing from Gradio users:
+
+* We've switched to modern technologies (like <a href="https://svelte.dev/" target="_blank">Svelte</a>) to build the Gradio frontend. We're seeing much smaller payloads and much faster page loads as a result! 
+
+* We've also embranced a much cleaner design that will allow Gradio demos to fit in visually in more settings (such as being embedded in blog posts). 
+
+<img class="max-w-full mx-auto my-6" style="width: 54rem" src="/blog/assets/68_gradio_blocks/lion.png">
```
Comment (nateraw) [inline gradio-blocks.md:40]:
```suggestion
<img class="max-w-full mx-auto my-6" style="width: 54rem" src="/blog/assets/68_gradio_blocks/lion.jpg">
```

File: gradio-blocks.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+                image_input = gr.Image()
+                image_output = gr.Image()
+            button = gr.Button("Flip")
+            # this demo runs whenever the button is clicked
+            button.click(flip_image, inputs=image_input, outputs=image_output)
+    
+demo.launch()
+```
+
+Once you run `launch()`, the following demo runs on a port:
```
[the remark this answers is not in the document]
Comment (abidlabs) [reply gradio-blocks.md:98]: Nope, will fix!

File: gradio-blocks.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
+---
+title: "Gradio 3.0 is Out!"
+thumbnail: /blog/assets/42_gradio_joins_hf/thumbnail.png
```
[the remark this answers is not in the document]
Comment (abidlabs) [reply gradio-blocks.md:3]: Done

File: gradio-blocks.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+More than 600,000 ML demos have been built with the Gradio library since its first version in 2019, and today, we are thrilled to announce **Gradio 3.0**: a ground-up redesign of the Gradio library 🥳
+
+
+### What's New in Gradio 3.0?
+
+🔥 A complete redesign of the frontend, based on the feedback we're hearing from Gradio users:
+
+* We've switched to modern technologies (like <a href="https://svelte.dev/" target="_blank">Svelte</a>) to build the Gradio frontend. We're seeing much smaller payloads and much faster page loads as a result! 
+
+* We've also embranced a much cleaner design that will allow Gradio demos to fit in visually in more settings (such as being embedded in blog posts).
```
[the remark this answers is not in the document]
Comment (abidlabs) [reply gradio-blocks.md:38]: Done

**Comment by abidlabs** [comment]
Great, thanks Nate! I do want to draw attention to the Blocks Party, so maybe we can leave the thumbnail. I'll provide context in the announcement as you suggested.

**Pull request merged**